test-resources/java/feature_flag_system_1/treated/input/XPFlagCleanerPositiveCases.java

```diff
@@ -71,14 +71,14 @@ public void conditional_contains_stale_flag() {
     if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
       System.out.println("Hello World");
     }
-    if (!experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
+    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG) == false) {
       System.out.println("Hi World");
     }
   }
 
   public void conditional_with_else_contains_stale_flag() {
 
-    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) {
+    if (experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG) == true) {
```

Comment:
Can we also have a case for:

```
if ((experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG)) == true) { ... }
```

I wonder how sensitive are those `#eq?` and `#not-eq?` operators, and what will happen here if any of the intermediate states is something like `((true) = true)`. Will it be able to resolve it to `true`, or will the second rule apply?

Reply:
I see. Nice catch. Let me think about it and get back to u